color/src/color.rs

```diff
@@ -302,10 +304,12 @@ impl<CS: ColorSpace> AlphaColor<CS> {
     #[must_use]
     pub fn map_lightness(self, f: impl Fn(f32) -> f32) -> Self {
```

Comment:
Worth a small note in the doc comment that this differs from the range for Lab / Lch… it is stated, but not directly enough, I think.